src/ngraph/pass/opset1_upgrade.cpp

```diff
@@ -547,6 +547,35 @@ namespace
         return true;
     }
 
+    bool op_cast(shared_ptr<op::Split> node)
+    {
+        const auto& splits_vec = node->get_splits();
+        const auto first_elem = splits_vec.front();
+
+        const bool split_evenly =
+            std::all_of(splits_vec.begin(), splits_vec.end(), [first_elem](const size_t split) {
+                return split == first_elem;
+            });
+
+        std::shared_ptr<Node> replacement_node;
+        if (split_evenly)
```

Comment:
This looks like it maybe should be the other way around (`if`/`else`).

Reply:
done